.github/workflows/package.yml

```diff
@@ -21,4 +31,9 @@ jobs:
       - name: Build electron app
         run: npm run build:electron
       - name: build package
-        run: npx electron-builder build --x64 --arm64 --publish never
+        run: npx electron-builder build --x64 --arm64 --publish always
+      - name: Upload built packages
```

Comment:
This is for a checking purpose in each action such as https://github.com/appium/appium-inspector/actions/runs/11700110550. After confirming this works with regular release cycles in the release page, we can remove this.